contracts/extensions/BatchTradeExtension.sol

```diff
@@ -113,8 +105,6 @@ contract BatchTradeExtension is BaseGlobalExtension {
      * @param _delegatedManager     Instance of the DelegatedManager to initialize
      */
     function initializeExtension(IDelegatedManager _delegatedManager) external onlyOwnerAndValidManager(_delegatedManager) {
-        require(_delegatedManager.isPendingExtension(address(this)), "Extension must be pending");
```

Comment:
When we `_initializeExtension` we call `DelegatedManager.initializeExtension` which checks the extension status https://github.com/SetProtocol/set-v2-strategies/blob/00aa1ac34c465bd34dde8d997aa91052090c63ce/contracts/manager/DelegatedManager.sol#L219